CHANGELOG

```diff
@@ -12,6 +12,19 @@ devel
   after the other. However, inside each database, the collections of the
   database can be dumped in parallel using multiple threads.
 
+* added "name" property for indices
+
+  If a name is not specified on index creation, one will be auto-generated.
+
+* Under normal circumstances there should be no need to connect to a 
+  database server in a cluster with one of the client tools, and it is 
+  likely that any user operations carried out there with one of the client
+  tools may cause trouble. 
+
+  The client tools arangosh, arangodump and arangorestore will now emit 
+  a warning when connecting with them to a database server node in a cluster.
+>>>>>>> 0c6de0c6a806530f67f16ecc2f4c612893e5ae55
```

Comment:
leftover from unresolved merge conflict?

Reply:
That change was already outdated when you reviewed. I removed the stray `>>>>>` line in another commit afterwards.